lib/config/configuration.go

```diff
@@ -362,6 +362,9 @@ func ApplyFileConfig(fc *FileConfig, cfg *service.Config) error {
 			return trace.Wrap(err)
 		}
 	}
+
+	// apply AWS section
+	cfg.AWS.InjectTags = fc.AWS.InjectTags
```

Comment:
I would not make this specific to AWS. Most (all?) clouds have tags. We should have generic "native tag" concept.